src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -585,8 +585,16 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
   renderProductImage() {
     const { productData, arFileExists, multiImages } = this.state;
     const settings = {
-      dots: false,
-      infinite: true,
+      customPaging(i) {
```

Comment:
Can we disable customPaging if `multiImages` only has a single element? Just to prevent an experience like the following where a thumbnail of the single image is displayed with the original:

![image](https://user-images.githubusercontent.com/39800563/75157179-95424500-56e1-11ea-9ac2-6ad5d75e7b6d.png)